app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -96,6 +96,7 @@ class HomePresenter(
         if (state.tokens.isEmpty()) {
             initialLoadTokens()
         } else {
+            initializeActionButtons()
             handleUserTokensLoaded(state.tokens)
```

Comment:
If user is new and he has internet, he'll not see the action buttons?

Reply:
he will see them is everything is ok in initialLoadTokens

Reply:
I just checked that `initializeActionButtons` is being called on here.
If `initialLoadTokens` succeeded it doesn't set the buttons to adapter. 
Are you sure it will work?